linalg_ops/convolution/CMakeLists.txt

```diff
@@ -0,0 +1,149 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# TODO(scotttodd): add filtering here, in the helper functions, or in ctest to
+#                  choose which tests to compile and run
+
+set(_SIZES)
+list(APPEND _SIZES "large")
+list(APPEND _SIZES "medium")
+list(APPEND _SIZES "small")
+
+###############################################################################
+#
+# CPU - llvm-cpu on local-task, default flags.
+#
+###############################################################################
+
+
+set(_DTYPES)
+list(APPEND _DTYPES "f16_f16_f16")
+list(APPEND _DTYPES "f32_f32_f32")
```

Comment:
Is there a reason not to have i8_i8_i32 or f16_f16_f32? Would be good to have some type promotion tested on CPU as well. (although we probably can't meaningfully test winograd with i8, so you can leave that one out for winograd).